logbook-spring-boot-autoconfigure/src/main/java/org/zalando/logbook/autoconfigure/LogbookAutoConfiguration.java

```diff
@@ -161,7 +162,7 @@ public QueryFilter queryFilter() {
         final List<String> parameters = properties.getObfuscate().getParameters();
         return parameters.isEmpty() ?
                 QueryFilters.defaultValue() :
-                replaceQuery(new HashSet<>(parameters)::contains, "XXX");
+                replaceQuery(new HashSet<>(parameters)::contains, properties.getObfuscate().getReplacement());
```

Comment:
is it a good idea to make the replacement configurable? In theory we could replace it with everything then. Or at least we should test it with  any imaginable weird replacement. Any kind of whitespace, special characters, even code snippets maybe ;)